pkg/ingester/ingester.go

```diff
@@ -206,6 +206,9 @@ type Config struct {
 	UpdateIngesterOwnedSeries       bool          `yaml:"track_ingester_owned_series" category:"experimental"`
 	OwnedSeriesUpdateInterval       time.Duration `yaml:"owned_series_update_interval" category:"experimental"`
 
+	// TODO Make "false" by default, after IngestStorageConfig.Enabled is stable
```

Comment:
Please remove this comment for two reasons:
1. We can safely switch to false only once ingest storage will be the default (but it's very far from now)
2. We've a bunch of other config that will be changed once ingest storage will be the default, so we can address all of them later (in a far future...)